### PR TITLE
fix: fixes bp metadata desc issue and adds architecture desc attribute

### PR DESCRIPTION
**Pull request by g-awmalik**

### CODE REVIEW

File: cli/testdata/bpmetadata/md/list-content.md [hunk 1]
```diff
@@ -29,3 +29,7 @@ It creates projects and configures aspects like Shared VPC connectivity, IAM acc
 
 ### preDeploy
 To deploy this blueprint you must have an active billing account and billing permissions.
+
+### Architecture
+1- User requests are sent to the front end, which is deployed on two Cloud Run services as containers to support high scalability applications.
+2- The request then lands on the middle tier, which is the API layer that provides access to the backend. This is also deployed on Cloud Run for scalability and ease of deployment in multiple languages. This middleware is a Golang based API.
```
Comment (bharathkkb) [inline cli/testdata/bpmetadata/md/list-content.md:35]:
```suggestion
1- User requests are sent to the front end, which is deployed on two Cloud Run services as containers to support high scalability applications.

2- The request then lands on the middle tier, which is the API layer that provides access to the backend. This is also deployed on Cloud Run for scalability and ease of deployment in multiple languages. This middleware is a Golang based API.
```

File: cli/bpmetadata/types.go [hunk 2]
```diff
@@ -62,9 +62,10 @@ type BlueprintActuationTool struct {
 }
 
 type BlueprintDescription struct {
-	Tagline   string `json:",omitempty" yaml:",omitempty"`
-	Detailed  string `json:",omitempty" yaml:",omitempty"`
-	PreDeploy string `json:"preDeploy,omitempty" yaml:"preDeploy,omitempty"`
+	Tagline      string `json:",omitempty" yaml:",omitempty"`
+	Detailed     string `json:",omitempty" yaml:",omitempty"`
+	PreDeploy    string `json:"preDeploy,omitempty" yaml:"preDeploy,omitempty"`
+	Architecture string `json:"architecture,omitempty" yaml:"architecture,omitempty"`
```
Comment (bharathkkb) [inline cli/bpmetadata/types.go:68]: Should we make this a []string and require an ordered list in md?

File: cli/testdata/bpmetadata/md/list-content.md [hunk 3]
```diff
@@ -29,3 +29,7 @@ It creates projects and configures aspects like Shared VPC connectivity, IAM acc
 
 ### preDeploy
 To deploy this blueprint you must have an active billing account and billing permissions.
+
+### Architecture
```
Comment (bharathkkb) [inline cli/testdata/bpmetadata/md/list-content.md:33]: From a UX perspective, embedding the diagram here would let users correlate the diagram and the text.

**Review by bharathkkb — APPROVED** [review]
Thanks @g-awmalik some minor comments

File: cli/testdata/bpmetadata/md/list-content.md (hunk 1 above)
Comment (g-awmalik) [reply cli/testdata/bpmetadata/md/list-content.md:35]: I plan to change this to an ordered list implementation. Right now the content gets cut off if there's more than one `\n` detected.

File: cli/bpmetadata/types.go (hunk 2 above)
Comment (g-awmalik) [reply cli/bpmetadata/types.go:68]: Yes, once the implementation is changed, I will change this to an array

File: cli/testdata/bpmetadata/md/list-content.md (hunk 3 above)
Comment (g-awmalik) [reply cli/testdata/bpmetadata/md/list-content.md:33]: Yeah I thought about that. Again requires more changes but we should do it. I just wanted this initial fix to go out so we can share a better version of metadata with the partners.

File: cli/bpmetadata/types.go (hunk 2 above)
Comment (g-awmalik) [reply cli/bpmetadata/types.go]: Ended up changing it with this release.

File: cli/testdata/bpmetadata/md/list-content.md (hunk 1 above)
Comment (g-awmalik) [reply cli/testdata/bpmetadata/md/list-content.md]: Ignore this since now we're looking for an ordered list.

**Pull request merged**